box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxFolder.java

```diff
@@ -101,6 +102,34 @@ public static BoxFolder createFromId(String folderId) {
     }
 
     /**
+     * A convenience method that copies all properties of a folder except the item collection.
+     *
+     * @param folder Original folder to create a copy off of
+     * @return copy of a folder with an empty item collection
+     */
+    public static BoxFolder copyFolderWithNoItems(final BoxFolder folder) {
```

Comment:
Mostly I'm hesitant to move it to SdkUtils since it is logic specific to BoxFolder. I don't know if we'll want these types of methods more times in the future for other file types, but I'd be hesitant to create more and more util methods there if so.